.github/workflows/bazel_test.yml

```diff
@@ -1,14 +1,44 @@
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 name: Bazel Test
 
 on: [push, pull_request]
 
 jobs:
-  linux:
+  bazel_test:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
+      - name: Install deps
+        shell: bash
+        run: |
+          mkdir -p bin/clang10
+          cd bin/clang10
+          wget https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/clang+llvm-10.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz                                                                        
+          tar -xf clang+llvm-10.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz --strip-components 1  
+          export PATH=bin/clang10/bin:$PATH
       - name: Run all tests
         working-directory: ./
+        # Run all bazel test but only need to build //examples/...
+        # Add bazel test --config=clang //... --test_tag_filters=test
+        # in the future if any test rules are added
         run: |
-          bazel test //...
+          export PATH=bin/clang10/bin:$PATH
+          echo $PATH
+          ls bin/clang10/
+          clang --version
+          bazel build --verbose_failures --build_tag_filters=fuzz_test --config=asan-libfuzzer --action_env=PATH=bin/clang10/bin:$PATH --action_env=CC=clang --action_env=CXX=clang++ --action_env=LLVM_CONFIG=bin/clang10/bin/llvm-config --repo_env=LLVM_CONFIG=bin/clang10/bin/llvm-config --linkopt=-Lbin/clang10/lib --linkopt=-Wl,-rpath,bin/clang10/lib //examples/...
```

Comment:
I see a bunch of new arguments here, which increases the complexity of the setup. If this was experimental code to tinker with FDP support, perhaps we can revert to the simpler form if we're going to de-prioritize FDP?

Reply:
+1

Reply:
This is just for debugging the problem. I am reverting it to simpler form.